[model] Add stdlib header for exit(3) and daemon on NetBSD

diff --git a/include/model/main.hh b/include/model/main.hh
index 08e3543..1f719bb 100644
--- include/model/main.hh
+++ include/model/main.hh
@@ -3,6 +3,7 @@
 
 #include <iostream>
 #include <string>
+#include <stdlib.h>
 #include <unistd.h>
 
 #include <boost/system/system_error.hpp>

